sdk/core/core/src/az_span_seq.c

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span_seq.h>
+
```

Comment:
nit: remove line breaks

Reply:
@sergey-shandar this is for you :)   
I am thinking I will better make a PR against your branch.